incubation-process.md

```diff
@@ -87,19 +87,15 @@ Projects at the Adopted stage fit into the overall architecture of the OpenRail
 
 ### Criteria
 
-*The criteria for stage 3 are still a draft*
-
-*Examples from CNCF incubation process, to be adopted*
-
 * Document that it is being used successfully in production by at least three independent adopters which, in the TC’s judgement, are of adequate quality and scope.
+* Have a public list of project adopters for at least the primary repo (preferably documented in the project's publiccode.yml file). For a specification, have a list of adopters for the implementation(s) of the spec.
 * Have committers from at least two organizations.
-* Have achieved and maintained an OpenSSF Best Practices Badge.
-* Provide SBOMs for all release artifacts
+* The project has a documented track record of regular stable releases
+* The project has a documented process with guaranteed reaction times for releasing security updates
 * Have completed an independent and third party security audit with results published of similar scope and quality as this example which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before moving to the Adopted stage.
-* Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
-* Explicitly define the criteria, process and offboarding or emeritus conditions for project maintainers; or those who may interact with the CNCF on behalf of the project. The list of maintainers should preferably be stored in a MAINTAINERS.md file and audited at a minimum of an annual cadence.
-* Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
-* Please include a short one-page narrative based on the template for Adopted projects, no more than 500 words.
+* There is at least one organization which is able to offer commercial support for the project.
+* At least 7/10 score on OpenSSF Best Practices Badge.
```

Comment:
Maybe more ambitious would be better: 8/10. 10/10 probably is too tough to achieve. Not all criteria might actually be as impactful and the effort needs to stay reasonable.

Maybe we need to have a look at the actual criteria for the badge and see what we consider to be important and then see what score reflects that.

Reply:
Changed it to 8/10 for now. We still need to do a detailed analysis what this would mean in terms of concrete criteria. Maybe we should postpone that to the time when we have an actual project applying for stage 3 and then sort it out with the concrete example.